src/main/webapp/app/shared/sidebar/sidebar-card-large/sidebar-card-large.component.html

```diff
@@ -5,5 +5,6 @@
     (click)="emitStoreAndRefresh(sidebarItem.id)"
     [routerLinkActive]="'bg-selected border-selected'"
 >
-    <jhi-sidebar-card-item [sidebarType]="sidebarType" [sidebarItem]="sidebarItem" />
+    <!-- groupKey is used to determine the display format of channel names in different sidebar sections -->
+    <jhi-sidebar-card-item [groupKey]="groupKey" [sidebarType]="sidebarType" [sidebarItem]="sidebarItem" />
```

Comment:
need to be accessed with groupKey() after implementing signals
```suggestion
    <jhi-sidebar-card-item [groupKey]="groupKey()" [sidebarType]="sidebarType" [sidebarItem]="sidebarItem" />
```